recipes/sibeliaz/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2.2" %}
+{% set sha256 = "159716b619db8ddfe07ac5c048a6937aa193a9ddc9a0d80389242be4b873b343" %}
+
+package:
+  name: sibeliaz
+  version: {{ version }}
+
+source:
+  url: https://github.com/medvedevgroup/SibeliaZ/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - sibeliaz.patch
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - openmp
+  host:
+    - twopaco
+    - spoa 4.0.3-0
+    - tbb-devel
+    - maf2synteny
```

Comment:
You don't need any of these during run-time?

Reply:
Oops, thanks for the catch!